examples/osgi-test-example-bnd-workspace/org.osgi.test.example.player.test/bnd.bnd

```diff
@@ -0,0 +1,70 @@
+# Build dependencies 
+-buildpath: \
+	${junit},\
+	${mockito},\
+	${osgitest},\
+	osgi.annotation;version=latest,\
+	osgi.core;version=latest,\
+	junit-jupiter-api;version=latest,\
+	org.osgi.test.example.api;version=latest,\
+	org.apiguardian:apiguardian-api;version=latest,\
+	org.osgi.namespace.service;version=latest
+
+Bundle-Version: 0.0.0.${tstamp}
+
+# This is to run the Test
+-runfw: org.eclipse.osgi
+
+-runproperties: \
+	logback.configurationFile=${fileuri;${.}/logback.xml},\
+	org.apache.felix.http.host=localhost,\
+	org.osgi.service.http.port=*,\
+	org.osgi.framework.bootdelegation=sun.reflect,\
+	osgi.console=
+
+-runsystempackages: \
+	org.slf4j;version=1.7.25,\
+	org.slf4j.helpers;version=1.7.25,\
+	org.slf4j.spi;version=1.7.25
+	
+-runpath: \
+	ch.qos.logback.classic,\
+	ch.qos.logback.core,\
+	org.apache.felix.logback,\
+	slf4j.api
+	
+-runrequires: \
+	bnd.identity;id='junit-jupiter-engine',\
+	bnd.identity;id='junit-platform-launcher',\
+	bnd.identity;id='org.osgi.test.example.player.test'
```

Comment:
This should use a macro like the other examples. `id=${p}`

Reply:
now its `bnd.identity;id='${p}.test'`